### PR TITLE
Added Clear Feature

### PR DESCRIPTION
**Pull request by appcreatorguy**
Added the ability to clear the universe by pressing tab.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -243,6 +243,10 @@ def draw():
             planets.remove(planet)
         if planet.velocity[1] > 1000 or planet.velocity[1] < -1000:
             planets.remove(planet)
+    
+    if tab_down:
+        for planet in planets:
```
Comment (000Nobody) [inline main.py]: This could be changed to planets.clear() for simplicity

**Comment by 000Nobody** [comment]
This is a really good idea, thanks! For simplicity, I think that you could remove the for loop when removing planets and just replace it with `planets.clear()`. Also, you could remove the tab_down variable completely and instead just clear planets on KEYDOWN and get rid of the whole tab KEYUP check.

**Comment by appcreatorguy** [comment]
Ok, made those changes! I might work on some solar system presets or other things too!

**Comment by 000Nobody** [comment]
Looks good, thanks

**Pull request merged**